tests/unit/references/test_processors.py

```diff
@@ -120,3 +123,31 @@ def test_reference_builder_add_uid():
     }
 
     assert expected == rb.obj
+
+
+def test_refextract_authors():
+    author_strings = [
+        'Butler, D., Demarque, P., & Smith, H. A.',
+        'Cenko, S. B., Kasliwal, M. M., Perley, D. A., et al.',
+        'J. Kätlne et al.', # Also test some unicode cases.
+        u'J. Kätlne et al.',
+        'Hoaglin D. C., Mostellar F., Tukey J. W.',
+        'V.M. Zhuravlev, S.V. Chervon, and V.K. Shchigolev',
+        'Gómez R, Reilly P, Winicour J and Isaacson R'
+    ]
+
+    expected = [
+        ['Butler, D.', 'Demarque, P.', 'Smith, H. A.'],
+        ['Cenko, S. B.', 'Kasliwal, M. M.', 'Perley, D. A.'],
+        ['J. Kätlne'],
+        ['J. Kätlne'],
+        ['Hoaglin D. C.', 'Mostellar F.', 'Tukey J. W.'],
+        ['V.M. Zhuravlev', 'S.V. Chervon', 'V.K. Shchigolev'],
```

Comment:
Still, kind of the same. I wouldn't proceed with anything else for the moment.

* In matching I always used raw tokens without first name / last name info mostly because of these things being wrong
* In display it's the same thing for me :)